x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/config/ModelPlotConfig.java

```diff
@@ -67,7 +67,7 @@ public ModelPlotConfig(StreamInput in) throws IOException {
     public void writeTo(StreamOutput out) throws IOException {
         out.writeBoolean(enabled);
         out.writeOptionalString(terms);
-        if (out.getVersion().onOrAfter(Version.V_8_0_0)) {
+        if (out.getVersion().onOrAfter(Version.V_7_9_0)) {
```

Comment:
As above, you might as well just delete this line.

Reply:
Done.